core/roboconf-agent/src/test/java/net/roboconf/agent/internal/misc/AgentUtilsTest.java

```diff
@@ -271,4 +275,30 @@ public void testFindIpAddress_nforceDefaultNetworkInterface() throws Exception {
 		String expectedIp = InetAddress.getLocalHost().getHostAddress();
 		Assert.assertEquals( expectedIp, AgentUtils.findIpAddress( AgentConstants.DEFAULT_NETWORK_INTERFACE ));
 	}
+
+	@Test
+	public void testExecuteScriptResources() throws IOException, InterruptedException {
+
+		//prepare our resources
+		TestApplicationTemplate app = new TestApplicationTemplate();
+		Instance instance = app.getTomcatVm();
+
+		List<ExecutionResult> result1 = AgentUtils.executeScriptResources(instance, app.getName(), instance.getName());
+		Assert.assertEquals(0,result1.size());
+
+		File dir = InstanceHelpers.findInstanceDirectoryOnAgent( instance );
+		Utils.createDirectory( dir );
+		File script1 = new File( dir, "toto.script" );
```

Comment:
This only works on Linux system.
How does it work on Windows? You suggest you use JUnit assumtions.

```java
Assume( functionThatReturnsTrueIfWeAreOnLinux());
```